prereise/utility/generate_rural_shapefiles.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import geopandas as gpd
+
+from prereise.gather.const import abv2state
+from prereise.utility.shapefile import download_shapefiles
+
+
+def append_rural_areas_to_urban(states, urban_areas):
+    """Takes shapefiles of US States and US urban areas, processes the files,
+    and adds rural areas (areas that are not urban) to the geodataframe.
+
+    :param geopandas.geodataframe.GeoDataFrame states: US state outlines
+    :param geopandas.geodataframe.GeoDataFrame urban_areas: US urban area outlines
+    :return: (*geopandas.geodataframe.GeoDataFrame*) -- new gdf of urban and rural areas
+        includes a column indicating whether the area is urban or not
+    """
```

Comment:
I think this docstring should include something about the filtering to the contiguous 48 states, or that logic should be moved to a separate function.